lib/infoblox/resource/zone.rb

```diff
@@ -0,0 +1,8 @@
+module Infoblox 
+  class Zone < Resource
+    attr_accessor :fqdn 
+    remote_attr_accessor :address, :comment, :disable, :network_view, :extattrs
+    
+    wapi_object "zone"
```

Comment:
My bad. You are right. There are few starting with zone, I will update it to zone_auth after testing it in NIOS. Thanks for the quick catch :+1: